### PR TITLE
Update documentation with systemd

### PR DESCRIPTION
**Pull request by JegoYann**

### CODE REVIEW

File: web/src/assets/snippets/production/wakanda.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Service]
```
Comment (sclairet) [inline web/src/assets/snippets/production/wakanda.conf]:
```
User=wakanda
Group=wakanda
Restart=alway
```
comme ce sont déjà les valeur par défaut, je ne sais pas si ça vaut le coup de les mettre ici

File: web/src/assets/sections/production.html [hunk 2]
```diff
@@ -79,35 +79,33 @@ <h3>Linux</h3>
...
-	            <code>wakanda_opt_with_syslog</code>: Flag for routing the solution log via syslog. "1" for enabling syslog (recommended), "0" for disabling it
+	            <code>Group</code>: The UNIX group that Wakanda Server will be running as.
 	        </li>
 	        <li>
-	            <code>wakanda_run_as_user</code>: The UNIX user that Wakanda Server will be running as. The Wakanda Solution must be accessible from this user (read/write). If the logs are routed via syslog, the user must have write-access to the "/var/log/wakanda" folder (Note: Wakanda Server must not be running as "root").
-	        </li>
-	        <li>
-	            <code>wakanda_opt_pid</code>: PID file containing Wakanda Server's process id
+	            <code>User</code>: The UNIX user that Wakanda Server will be running as. The Wakanda Solution must be accessible from this user (read/write). If the logs are routed via syslog, the user must have write-access to the "/var/log/wakanda" folder (Note: Wakanda Server must not be running as "root").
 	        </li>
```
Comment (sclairet) [inline web/src/assets/sections/production.html]: j'ajouterai qu'il faut appeler `systemctl daemon-reload' pour que le fichier soit pris en compte.

File: web/src/assets/sections/production.html [hunk 3]
```diff
@@ -79,35 +79,33 @@ <h3>Linux</h3>
...
 	        <li>
 	            Get Wakanda service status:
 	            <br>           
-	            <code>$ sudo service wakanda status</code>
+	            <code>$ systemctl status wakanda.service</code>
 	        </li>
 	    </ul>
 	
-	    The <code>/etc/default/wakanda</code> file allows you to configure the following:
+	    You can create a <code>/etc/systemd/system/wakanda.service.d/wakanda.conf</code> to configure the service and call <code>systemctl daemon-reload</code>:
```
Comment (dartzon) [inline web/src/assets/sections/production.html]: Create a configuration file at <code>/etc/systemd/system/wakanda.service.d/wakanda.conf</code> to configure the service and call <code>systemctl daemon-reload</code> to reload it:

File: web/src/assets/sections/production.html [hunk 4]
```diff
@@ -79,35 +79,33 @@ <h3>Linux</h3>
...
 	    <ul>
 	        <li>
-	            <code>wakanda_opt_solution</code>: Full pathname of the Wakanda solution to launch when the service starts
+	            <code>ExecStart</code>: Full cmd line to run as a service with every parameters inside (server path, syslog, solution path, admin port, ...)
```
Comment (dartzon) [inline web/src/assets/sections/production.html]: <code>ExecStart</code>: Full path and the arguments (server path, syslog, solution path, admin port, ...) of the command to be executed as a service.

File: web/src/assets/sections/production.html [hunk 5]
```diff
@@ -79,35 +79,33 @@ <h3>Linux</h3>
...
 	        <li>
-	            <code>wakanda_opt_with_syslog</code>: Flag for routing the solution log via syslog. "1" for enabling syslog (recommended), "0" for disabling it
+	            <code>Group</code>: The UNIX group that Wakanda Server will be running as.
 	        </li>
 	        <li>
-	            <code>wakanda_run_as_user</code>: The UNIX user that Wakanda Server will be running as. The Wakanda Solution must be accessible from this user (read/write). If the logs are routed via syslog, the user must have write-access to the "/var/log/wakanda" folder (Note: Wakanda Server must not be running as "root").
-	        </li>
-	        <li>
-	            <code>wakanda_opt_pid</code>: PID file containing Wakanda Server's process id
+	            <code>User</code>: The UNIX user that Wakanda Server will be running as. The Wakanda Solution must be accessible from this user (read/write). If the logs are routed via syslog, the user must have write-access to the "/var/log/wakanda" folder (Note: Wakanda Server must not be running as "root").
```
Comment (dartzon) [inline web/src/assets/sections/production.html]: <code>User</code>: The UNIX user that the Wakanda Server will be running as. The provided user must have the required privileges to access the Wakanda solution both on read and write. If the logs are routed via syslog, the user must have write-access to the "/var/log/wakanda/" folder (Note: Wakanda Server must not be running as "root").

**Pull request merged**